### PR TITLE
Add support for BRL

### LINKED ISSUE #1: Add BRL offramp support to prototype
Add support for offramping BRL via nTokens to the spacewalk-mykobo-offramp prototype.

### PR DESCRIPTION
**Pull request by TorstenStueber**
Closes #1

### CODE REVIEW

File: spacewalk-mykobo-offramp/src/index.js [hunk 1]
```diff
@@ -172,19 +206,16 @@ async function sep24(token, sep24Url) {
...
 
-  if (status.withdraw_memo_type !== "text") {
-    throw new Error(`Unexpected offramp memo type: ${transaction.withdraw_memo_type}`);
-  }
-
   console.log("SEP-24 parameters received");
   return {
     amount: status.amount_in,
     memo: status.withdraw_memo,
+    memoType: status.withdraw_memo_type,
```
Comment (gianfra-t) [inline spacewalk-mykobo-offramp/src/index.js:213]: The response from Mykobo's `sep24` also returns this value right? (in this case `text`)

**Review by gianfra-t — COMMENTED** [review]
Since the issuer of BRL also implements the `sep10, sep24` flow, the step for sending the actual tokens to the specified Stellar account for off-ramp is exactly the same also? only changing how the memo is defined? For instance, no different time constrains, etc.

File: spacewalk-mykobo-offramp/src/index.js (hunk 1 above)
Comment (TorstenStueber) [reply spacewalk-mykobo-offramp/src/index.js:213]: Yes, it does. See the [previous code](https://github.com/pendulum-chain/prototypes/pull/2/files#diff-565adf58deee5d36e2095ab56e349fbe78b45bcc7e787044c735d6bf1b904f17L175-L177) where I expected this to be `text` (I was actually surprised that nTokens uses a different memo format but this is allowed by the spec).

**Comment by TorstenStueber** [comment]
> Since the issuer of BRL also implements the `sep10, sep24` flow, the step for sending the actual tokens to the specified Stellar account for off-ramp is exactly the same also? only changing how the memo is defined? For instance, no different time constrains, etc.

Yes, the rest works the same.

**Review by gianfra-t — APPROVED** [review]
It looks good to me!

**Pull request merged**